src/main/java/org/thoughtcrime/securesms/qr/BackupReceiverFragment.java

```diff
@@ -84,7 +84,8 @@ public void handleEvent(@NonNull DcEvent event) {
             } else if (permille < 1000) {
                 percent = permille/10;
                 percentMax = 100;
-                statusLineText = getString(R.string.transferring) + String.format(Util.getLocale(), " %d%%", percent);
+                String formattedPercent = percent > 0 ? String.format(Util.getLocale(), " %d%%", percent) : "";
+                statusLineText = getString(R.string.transferring) + formattedPercent;
```

Comment:
btw, not related to this PR but this string concatenation is not good, for RTL languages it will not be mirrored properly :thinking:

Reply:
The good news is that it's not a sentence, so that it's not that bad if the order is wrong for RTL messages. I.e. "10% Transferring..." and "Transferring... 10%" is both right.